components/list/list-item-drag-drop-mixin.js

```diff
@@ -171,8 +171,19 @@ export const ListItemDragDropMixin = superclass => class extends superclass {
 	}
 
 	_onDragTargetClick(e) {
-		this.shadowRoot.querySelector(`#${this._itemDragId}`).activateKeyboardMode();
+		if (this._keyboardActiveOnNextClick) {
+			this.shadowRoot.querySelector(`#${this._itemDragId}`).activateKeyboardMode();
+		} else {
+			this.shadowRoot.querySelector(`#${this._itemDragId}`).focus();
```

Comment:
What is the user is trying to click on another focusable element in the list-item?  Seems like this would style focus away unexpectedly.

Reply:
Other focusable items will always appear in front because of the layered grid